jobdispatcher/src/main/java/com/firebase/jobdispatcher/SimpleJobService.java

```diff
@@ -18,7 +18,7 @@
 
 import android.os.AsyncTask;
 import android.support.annotation.CallSuper;
-// import android.support.annotation.GuardedBy;
```

Comment:
Sorry, still need to leave this in for the time being (pending some internal fixes currently in progress)